src/index.html

```diff
@@ -16,21 +16,28 @@
                 <li class="Subcribe">Subscribe</li>
             </ul>
         </nav>
+        <nav class="counter">
+            <h3> Movies (<span id="movie-counter"></span>)</h3>
+            <h3>Favourite</h3>
+          </nav>
     </header>
 
     <section class="main">
+        <div class="alert"></div>
+
+
         <div class="main-wrapper">
 
         </div>
     </section>
     <footer>
         <div class="designers">
           <p class="name"></p>
```

Comment:
Hello @frempongdev, thank you for helping me improve my source code. I'm sorry that line of code is where we render our modal popup. Removing it would mean our popup will not display.